apps/volume/app.js

```diff
@@ -19,14 +20,14 @@ module.exports = function (activity) {
     defaultVolume = 30
   }
 
-  function speakAndExit (text) {
+  function speakAndExit (ctx, text) {
     var ismuted = AudioManager.isMuted()
     if (ismuted) {
       AudioManager.setMute(false)
     }
     return activity.tts.speak(text).then(() => {
       if (ismuted) { AudioManager.setMute(true) }
-      return activity.exit()
+      return ctx.exit()
     })
   }
```

Comment:
I think this library should contain the speak/play/light.

Reply:
What shape of api would you propose?

Reply:
Standard (Experimental) Web Speech Synthesis API has much more complexity. I would suggest not mix those changes with this PR.